From c76f4dc577e7d68c8f581ec31fed4e14d4db31b2 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jose.dapena@lge.com>
Date: Fri, 5 Apr 2019 14:15:23 +0200
Subject: omnibox: AutocompleteMatch constructor cannot be noexcept default

The default constructor of AutocompleteMatch cannot be set as "noexcept"
as there are members that are not noexcept.

Bug: 819294
Change-Id: I8714f2c6352a3292bdebdc3aed9790270e49c580

diff --git a/components/omnibox/browser/autocomplete_match.cc b/components/omnibox/browser/autocomplete_match.cc
index 0053538036f8..acc9af1af3a4 100644
--- a/components/omnibox/browser/autocomplete_match.cc
+++ b/components/omnibox/browser/autocomplete_match.cc
@@ -154,8 +154,7 @@ AutocompleteMatch::AutocompleteMatch(const AutocompleteMatch& match)
       additional_info(match.additional_info),
       duplicate_matches(match.duplicate_matches) {}
 
-AutocompleteMatch::AutocompleteMatch(AutocompleteMatch&& match) noexcept =
-    default;
+AutocompleteMatch::AutocompleteMatch(AutocompleteMatch&& match) = default;
 
 AutocompleteMatch::~AutocompleteMatch() {
 }
diff --git a/components/omnibox/browser/autocomplete_match.h b/components/omnibox/browser/autocomplete_match.h
index edd4a078676a..4de013b44242 100644
--- a/components/omnibox/browser/autocomplete_match.h
+++ b/components/omnibox/browser/autocomplete_match.h
@@ -143,7 +143,7 @@ struct AutocompleteMatch {
                     bool deletable,
                     Type type);
   AutocompleteMatch(const AutocompleteMatch& match);
-  AutocompleteMatch(AutocompleteMatch&& match) noexcept;
+  AutocompleteMatch(AutocompleteMatch&& match);
   ~AutocompleteMatch();
 
   AutocompleteMatch& operator=(const AutocompleteMatch& match);
-- 
2.19.1

